test/python/pulse/test_pulse_lib.py

```diff
@@ -147,32 +147,6 @@ def test_construction(self):
         Sech(duration=50, amp=0.5, sigma=10, zero_ends=False)
         SechDeriv(duration=50, amp=0.5, sigma=10)
 
-    # This test should be removed once deprecation of complex amp is completed.
```

Comment:
This file can be also reverted because this is done by https://github.com/Qiskit/qiskit/pull/11403